op-service/txmgr/txmgr.go

```diff
@@ -797,14 +797,30 @@ func (m *SimpleTxManager) increaseGasPrice(ctx context.Context, tx *types.Transa
 	}
 
 	// Re-estimate gaslimit in case things have changed or a previous gaslimit estimate was wrong
-	gas, err := m.backend.EstimateGas(ctx, ethereum.CallMsg{
+	callMsg := ethereum.CallMsg{
 		From:      m.cfg.From,
 		To:        tx.To(),
 		GasTipCap: bumpedTip,
 		GasFeeCap: bumpedFee,
 		Data:      tx.Data(),
 		Value:     tx.Value(),
-	})
+	}
+	var bumpedBlobFee *big.Int
```

Comment:
Nit: could this var be moved inside the scope of the following `if` block? That is, if we don't need it up at this level of scope.

Reply:
It is used [here](https://github.com/zhiqiangxu/optimism/blob/5df2ee636c550e8942dd56553fc9255830ae5a65/op-service/txmgr/txmgr.go#L857), so can't be moved inside `if`.